core/unix/rseq_linux.c

```diff
@@ -229,7 +229,7 @@ static void
 rseq_clear_tls_ptr(dcontext_t *dcontext)
 {
     ASSERT(rseq_tls_offset != 0);
-    byte *base = get_segment_base(LIB_SEG_TLS);
+    byte *base = get_app_segment_base(LIB_SEG_TLS);
```

Comment:
How did this work before if it was getting the private base instead of the app base?

Reply:
Not sure. Probably wasn't a fatal error before.